docassemble/AssemblyLine/al_general.py

```diff
@@ -474,6 +475,8 @@ def init(self, *pargs, **kwargs):
             self.initializeAttribute("other_addresses", ALAddressList)
         if not hasattr(self, "mailing_address"):
             self.initializeAttribute("mailing_address", ALAddress)
+        if not hasattr(self, "service_address"):
+            self.initializeAttribute("service_address", ALAddress)
```

Comment:
I think there might also need to be an object block in the yml file, just
```yml
objects:
  - x.service_address: ALAddress
```

I've been bitten a few times when not putting stuff like that there, and breaking existing sessions of interviews, even if that's not likely to happen in this case